test/http.spec.js

```diff
@@ -0,0 +1,29 @@
+import nock from 'nock';
+import path from 'path';
+import downloadFile from '../src/download-file';
+import fs from 'fs';
+import expect from 'expect';
+
+nock('http://local.foo.com')
```

Comment:
I'm suggesting helpers to avoid repetition if and only if things start to get hairy. :)